packages/cli/src/tokens/build/utils/entryfile.ts

```diff
@@ -4,16 +4,53 @@ import glob from 'fast-glob';
 import fs from 'fs-extra';
 import * as R from 'ramda';
 
-const sortLightmodeFirst = R.sortWith<string>([R.descend(R.includes('light')), R.descend(R.includes('secondary'))]);
+/**
+ * Defines a sort order for the sections of the entry CSS file.
+ * This ensures a deterministic order, whereas earlier this was nondeterministic
+ */
+// biome-ignore format: keep array as one line per item
+const sortOrder = [
+  'color-mode/light',
+  'typography/secondary',
+  'semantic',
+  'color-mode/dark',
+  'color-mode/contrast',
+  'typography/primary',
+];
```

Comment:
This sort order keeps the brand css files unchanged from the existing commited files when running `yarn build:tokens`. But is there a reason for this exact ordering? The old sorting function would be equivalent to this:

```
const sortOrder = [
  'color-mode/light',
  'typography/secondary',
];
```
That is to say, these two would always come first, and then the rest in an unspecified — but **usually** the same — order.

However, wouldn't it make more sense for the things that are added to `:root` -- `color-mode/light`, `typography/primary`, and `semantic`, to be first, and then other color modes, then `typography/secondary`?

Any thoughts @Barsnes & @eirikbacker

Reply:
Suggest we create an issue of this and can as Mickey when he is back, and merge the rest of the changes - nice work 👏 🫶

Reply:
https://github.com/digdir/designsystemet/issues/2562